flow-typed/jest.js

```diff
@@ -26,7 +26,7 @@ type JestMockFn<TArguments: $ReadOnlyArray<*>, TReturn> = {
      * An array that contains all the object results that have been
      * returned by this mock function call
      */
-    results: Array<{ isThrow: boolean, value: TReturn }>,
```

Comment:
seems like your editor autoformatted these files. can you make sure they're excluded from prettier and revert these formatting changes?

Reply:
added .prettierignore and reverted changes in the flow-typed directory